conmon-rs/server/src/streams.rs

```diff
@@ -149,6 +173,8 @@ mod tests {
 
         assert_eq!(msg_string(msg)?, expected);
 
+        token.cancel();
```

Comment:
hmm. this one can too...

Reply:
same as above, the unit test has no entity paying attention to the command, so nothing else would call cancel()